### PR TITLE
Add option to remove uid from CSVLogger file path.

### LINKED ISSUE #59: Non-Mandatory CSVLogger UID (or custom UID)?
`class CSVLogger` necessarily adds a mandatory uid:

`directory = paths.process_path(directory, 'logs', label, add_uid=True)`

Would adding an argument to the `CSVLogger` init function so as to disable uids be possible? It'd look like:

```
class CSVLogger(base.Logger):
  """Standard CSV logger."""

  _open = open

  def __init__(self,
               directory: str = '~/acme',
               label: str = '',
               time_delta: float = 0.,
               add_uid=True,
):

    directory = paths.process_path(directory, 'logs', label, add_uid=add_uid)
```

### PR DESCRIPTION
**Pull request by ethanluoyc**
Fixes #59

### CODE REVIEW

**Review by sinopalnikov — CHANGES_REQUESTED** [review]
Looks good, thanks! Could you add a simple unit test covering this change?

**Comment by ethanluoyc** [comment]
> Looks good, thanks! Could you add a simple unit test covering this change?

I wanted to do that with absl parameterized tests. Is it ok if I change the base test case to inherit from that?

**Comment by sinopalnikov** [comment]
> > Looks good, thanks! Could you add a simple unit test covering this change?
> 
> I wanted to do that with absl parameterized tests. Is it ok if I change the base test case to inherit from that?

Yes, sure!

**Comment by ethanluoyc** [comment]
@sinopalnikov Done.

An unrelated note. Is it possible for the maintainers to add a `.gitignore` file to the repo? I have to locally exclude the __pycache__ and package files for my local development and it would be useful if there is a project-wide .gitignore set up. Another thing that would be useful is to add a pylintrc that can be used so that external contributors can lint their PRs. I believe using the one found [here](https://google.github.io/styleguide/pyguide.html) with some tweaks would suffice.

File: acme/utils/loggers/csv_test.py [hunk 1]
```diff
@@ -55,7 +56,11 @@ def test_logging_input_is_directory(self):
         outputs.append(dict(row))
     self.assertEqual(outputs, inputs)
 
-  def test_logging_input_is_file(self):
+  @parameterized.parameters(
+    {'add_uid': True},
+    {'add_uid': False},
+  )
```
Comment (sinopalnikov) [inline acme/utils/loggers/csv_test.py:62]:
```suggestion
  @parameterized.parameters(True, False)
```

**Review by sinopalnikov — CHANGES_REQUESTED** [review]
Could you simplify the parameterized annotation?

**Comment by sinopalnikov** [comment]
> An unrelated note. Is it possible for the maintainers to add a `.gitignore` file to the repo? I have to locally exclude the **pycache** and package files for my local development and it would be useful if there is a project-wide .gitignore set up. Another thing that would be useful is to add a pylintrc that can be used so that external contributors can lint their PRs. I believe using the one found [here](https://google.github.io/styleguide/pyguide.html) with some tweaks would suffice.

Yep, we'll look into adding them shortly.

**Comment by sinopalnikov** [comment]
ftr `.gitignore` and `.pylintrc` are now added:
https://github.com/deepmind/acme/commit/a6b4162701542ed08b0b911ffac7c69fcb1bb3c7

**Comment by ethanluoyc** [comment]
@sinopalnikov I have applied the review changes.

**Review by sinopalnikov — APPROVED** [review]
All good, thanks!

**Pull request merged**